src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/CppCodegenNodeFactory.cs

```diff
@@ -38,5 +44,33 @@ protected override ISymbolNode CreateReadyToRunHelperNode(Tuple<ReadyToRunHelper
             // TODO: this is wrong: this returns an assembly stub node
             return new ReadyToRunHelperNode(helperCall.Item1, helperCall.Item2);
         }
+
+        private void AddWellKnownType(WellKnownType wellKnownType, DependencyAnalysisFramework.DependencyAnalyzerBase<NodeFactory> graph)
+        {
+            var type = TypeSystemContext.GetWellKnownType(wellKnownType);
+            var typeNode = ConstructedTypeSymbol(type);
+            graph.AddRoot(typeNode, "Enables CPP codegen");
+        }
+        private void AddWellKnownTypes(DependencyAnalysisFramework.DependencyAnalyzerBase<NodeFactory> graph)
+        {
+
+            AddWellKnownType(WellKnownType.Void, graph);
```

Comment:
Can you put a TODO on line 56 (the empty line) to unify the list here with the list in `CppWriter`? This should be in a central location but I don't have time to think about where.